modules/microshift-preparing-to-make-app-rpms.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-preparing-to-make-app-rpms_{context}"]
+= Preparing to make application RPMs
+
+To build your own RPMs, choose a tool of your choice, such as the the `rpmbuild` tool, and initialize the RPM build tree in your home directory. The following is an example procedure. As long as your RPMs are accessible to Image Builder, you can use the method you prefer to build the application RPMs.
+
+.Prerequisites
+
+* You have set up a {op-system-ostree-first} {op-system-version} build host that meets the Image Builder system requirements.
+* You have root access to the host.
+
+.Procedure
+
+. Install the `rpmbuild` tool and create the yum repository for it by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install rpmdevtools rpmlint yum-utils createrepo
+----
+
+. Create the file tree you need to build RPM packages by running the following command:
++
+[source,terminal]
+----
+$ sudo rpmdev-setuptree
+----
+
+.Verification
+
+. List the directories to confirm creation by running the following command:
++
+[source,terminal]
+----
+$ tree ~/rpmbuild
```

Comment:
The tree command is an extra package that may need to be installed. I did not have it immediately available in RHEL 9.2 install

Reply:
For the purpose of verifying the directory structure was created 'ls ~/rpmbuild' would provide the same information.

$ ls ~/rpmbuild/
BUILD  RPMS  SOURCES  SPECS  SRPMS